package.json

```diff
@@ -10,6 +10,9 @@
   },
   "private": true,
   "dependencies": {
+    "@firebase/app": "^0.8.2",
```

Comment:
Will this (at some point) replace the `firebase-admin` dependency?

Reply:
I don't believe it will. As of now the sole reason for the addition of this dependency is to initialize a firebase app instance to pass into the getAuth method to initialize the firebase/auth instance. My understanding is that this particular step is not needed within a browser since a default instance is available there.